sdk/include/psdk/winbase.h

```diff
@@ -1362,11 +1362,11 @@ typedef enum _COMPUTER_NAME_FORMAT {
 	ComputerNamePhysicalDnsFullyQualified,
 	ComputerNameMax
 } COMPUTER_NAME_FORMAT;
-#endif /* (_WIN32_WINNT >= 0x0500) */
-
-#if (_WIN32_WINNT >= 0x0600)
+ /* (_WIN32_WINNT >= 0x0500) */
+//Modified by Freakston for Syzkaller
```

Comment:
If you're building it within our build system... could you just define _WIN32_WINNT to be a higher version and link to kernel32_vista?